MaterialLib/SolidModels/MFront/MFront.cpp

```diff
@@ -324,7 +347,10 @@ MFront<DisplacementDim>::integrateStress(
     {
         sigma[i] = behaviour_data.s1.thermodynamic_forces[i];
     }
-    sigma = MFrontToOGS(sigma);
+    sigma = Q.template topLeftCorner<
+                KelvinVectorDimensions<DisplacementDim>::value,
+                KelvinVectorDimensions<DisplacementDim>::value>() *
+            MFrontToOGS(sigma);
```

Comment:
We rotate sigma back into OGS coordinate system, so epsilon and sigma will be output in xyz. All MFront internal variables (ElasticStrain for example) will be output in the local coordinate system. in the _behaviour.symmetry != mgis::behaviour::Behaviour::Symmetry::ISOTROPIC case we could output a warning that his is the case and users have to be careful. Having the local coordinate system as an output in ParaView for use in transformations would be convenient (but not necessary).

Reply:
A todo at the place where the rotation should happen added.